docs/setup/config.md

```diff
@@ -85,6 +85,8 @@ The default fraction for managed memory can be adjusted using the `taskmanager.m
 
 - `taskmanager.memory.preallocate`: Can be either of `true` or `false`. Specifies whether task managers should allocate all managed memory when starting up. (DEFAULT: false)
 
+- `taskmanager.runtime.large-record-handler`: Whether to use the LargeRecordHandler when spilling. This feature is experimental. (DEFAULT: false)
```

Comment:
Not really, I'm afraid. I just recently disabled it by default because of the known issues.

What I can gather from the code is that it is a separate sort buffer that is intended for very large records. It is not obvious from the code what "very large records" are, however.

The known problem is that key serialization does not work correctly if the user specified a custom type or if Scala types are used because the `TypeAnalyzer` is used in the `LargeRecordHandler` to get a `TypeInformation` on the fly.